adapters/huaweiads/huaweiads.go

```diff
@@ -337,19 +400,21 @@ func getHeaders(huaweiAdsImpExt *openrtb_ext.ExtImpHuaweiAds, request *openrtb2.
 }
 
 // getHuaweiAdsReqJson: get body json for HuaweiAds request
-func getHuaweiAdsReqJson(request *huaweiAdsRequest, openRTBRequest *openrtb2.BidRequest, huaweiAdsImpExt *openrtb_ext.ExtImpHuaweiAds) error {
+func getHuaweiAdsReqJson(request *huaweiAdsRequest, openRTBRequest *openrtb2.BidRequest,
+	huaweiAdsImpExt *openrtb_ext.ExtImpHuaweiAds, extraInfo ExtraInfo) (string, error) {
 	request.Version = huaweiAdxApiVersion
 	var err error
-	if err = getHuaweiAdsReqAppInfo(request, openRTBRequest); err != nil {
-		return err
+	var countryCode string
+	if countryCode, err = getHuaweiAdsReqAppInfo(request, openRTBRequest, extraInfo); err != nil {
```

Comment:
you can avoid having to declare `err` and `countryCode` on separate lines by using
`if countryCode, err := getHuaweiAdsReqAppInfo(...`

This is just a stylistic quibble, if you prefer your current construction, that is valid as well.

Reply:
Thank you，done